client/post-editor/controller.js

```diff
@@ -172,21 +173,41 @@ async function maybeCalypsoifyGutenberg( context, next ) {
 
 	const state = context.store.getState();
 	const siteId = getSelectedSiteId( state );
-	const postType = determinePostType( context );
-	const postId = getPostID( context );
 
 	if (
-		( isCalypsoifyGutenbergEnabled( state, siteId ) ||
-			isGutenlypsoEnabled( state, siteId ) ||
-			isEnabled( 'jetpack/gutenframe' ) ) &&
-		'gutenberg' === getSelectedEditor( state, siteId )
+		! isCalypsoifyGutenbergEnabled( state, siteId ) &&
+		! isGutenlypsoEnabled( state, siteId ) &&
+		! isEnabled( 'jetpack/gutenframe' )
 	) {
-		let url = getEditorUrl( state, siteId, postId, postType );
-		// pass along parameters, for example press-this
-		url = addQueryArgs( context.query, url );
-		return window.location.replace( url );
+		return next();
+	}
+
+	if ( 'gutenberg' !== getSelectedEditor( state, siteId ) ) {
+		return next();
 	}
-	next();
+
+	const postType = determinePostType( context );
+	const postId = getPostID( context );
+
+	if ( false === isJetpackMinimumVersion( state, siteId, '7.3-alpha' ) ) {
```

Comment:
This needs to be a falsey check; `isJetpackMinimumVersion` [can return](https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/selectors.js#L235) `null` if the site isn't a Jetpack site, or the site option doesn't exist.

```suggestion
	if ( ! isJetpackMinimumVersion( state, siteId, '7.3-alpha' ) ) {
```

Reply:
If it isn't a Jetpack site we don't want the condition to be true, do we?

Reply:
Oops! Sorry, you're right, carry on :)